docs/tutorials/lrs/authentication/index.md

```diff
@@ -0,0 +1,13 @@
+
+# Authentication
+
+The API server supports the following authentication methods:
+
+- [HTTP basic authentication](basic.md)
+- [OpenID Connect authentication](oidc.md) on top of OAuth2.0
+
+One, the other or both can be enabled for Ralph LRS using the environment variable `RALPH_RUNSERVER_AUTH_BACKEND`:
+
+```bash
```

Comment:
In the `authentication` section, we still have not talk about the `.env` file, and we're passing environment variables through the `environment` attribute.
I would like to add a section focusing on the environment variables, but I think it would be after the `Backends` section.

Reply:
Alright, it makes sense !